### PR TITLE
Add custom endpoint for Klarna Prod and SB Environments.

### PR DESCRIPTION
**Pull request by syed-hasnain-cko**

### CODE REVIEW

**Review by philippe-leonhardt-cko — APPROVED** [review]

File: src/Library/Controller.php [hunk 1]
```diff
@@ -95,4 +95,14 @@ protected function response(HttpHandler $response, $qualified, $mode = HttpHandl
...
+
+    /**
+     * Return if the environment is Sandbox
+     *
+     * @return int getSandbox
+     */
+    public static function getSandbox()
+    {
+        return $this->configuration->getSandbox();
+    }
```
Comment (aquila-freitas-cko) [inline src/Library/Controller.php:107]: Hey mate, this code is not being used, this was a block from #82 . Could you remove this block, please?

**Review by aquila-freitas-cko — CHANGES_REQUESTED** [review]
Hey mate, minor comment from me but looks good overall.

File: src/Library/Controller.php (hunk 1 above)
Comment (syed-hasnain-cko) [reply src/Library/Controller.php:107]: Sure, will do.

**Review by aquila-freitas-cko — APPROVED** [review]
All good, well done!

**Pull request merged**